src/platform/android/java/chip/platform/NsdManagerServiceResolver.java

```diff
@@ -37,15 +40,27 @@ public class NsdManagerServiceResolver implements ServiceResolver {
   private Handler mainThreadHandler;
   private List<NsdManager.RegistrationListener> registrationListeners = new ArrayList<>();
   private final CopyOnWriteArrayList<String> mMFServiceName = new CopyOnWriteArrayList<>();
+  private final NsdManagerResolverAvailState nsdManagerResolverAvailState;
```

Comment:
We have a constructor that can set this to null. Can we tag this as nullable?

Reply:
Done.